app/views/search/all_content.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-2">
+m <div class="col-md-2">
```

Comment:
Seems like you added this `m` by mistake. Kindly edit the file and remove the `m` at the beginning of the file and commit.
you can find it here https://github.com/Annysah/plots2/blob/patch-6/app/views/search/all_content.html.erb.